addons/default/visiosoft/advs-module/resources/lang/ar/message.php

```diff
@@ -10,13 +10,13 @@
     'updated_category_msg' => 'تم تعديل الفئة بنجاح.',
     'max_ad_limit' => [
         'title' => "حد إعلانك المجاني خارج.",
-        'continueBtn' => "Show Ads",
-        'link' => "/profile",
+        'continueBtn' => "إظهار الإعلانات",
+        'link' => "/الملف الشخصي",
```

Comment:
@diashalabi are we using this link as html link 

@ozcan-durak

Reply:
this link is to redirect to the profile if you reached max ads. we shouldn't translate it

Reply:
We can edit the section and merge it @fatihalp or you may ignore from crowdin if possible

Reply:
if it's not translable put it in code not lang files.  We can't guarantee that will not translate. Even some customers translating. translations can't break system @diashalabi

Reply:
when I saw it I removed it but then it broke the system so I put it back, we can open an issue for it